pkgs/by-name/lu/lunarvim/package.nix

```
@@ -0,0 +1,128 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, fetchurl
+, makeWrapper
+, cargo
+, curl
+, fd
+, fzf
+, git
+, gnumake
+, gnused
+, gnutar
+, gzip
+, neovim
+, nodejs
+, nodePackages
+, ripgrep
+, tree-sitter
+, unzip
+, nvimAlias ? false
+, viAlias ? false
+, vimAlias ? false
+, globalConfig ? ""
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "lunarvim";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "LunarVim";
+    repo = "LunarVim";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-z1Cw3wGpFDmlrAIy7rrjlMtzcW7a6HWSjI+asEDcGPA=";
+  };
+
+  # Pull in the fix for Nerd Fonts until the next release
+  patches = [(
+    fetchurl {
+      url = "https://github.com/LunarVim/LunarVim/commit/d187cbd03fbc8bd1b59250869e0e325518bf8798.patch";
+      sha256 = "00677d3d5a4882d7ee5709e0494a5d8f7c58cea8bdcf467ce59222f9cc493366";
```

```suggestion
      hash = "sha256-BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB=";
```
Could you please use SRI hash?
https://nixos.org/manual/nixpkgs/stable/#fetchurl